app/modules/intelligence/agents/chat_agents/tool_helpers.py

```
@@ -600,38 +915,131 @@ def get_tool_call_info_content(tool_name: str, args: Dict[str, Any]) -> str:
+                kind_info = f" (kind: {kind})" if kind else ""
+                return f"-> searching code structure{kind_info}\n"
+        case "search_bash":
+            command = args.get("command", "")
+            working_dir = args.get("working_directory")
+            if command:
+                dir_info = f" in directory '{working_dir}'" if working_dir else ""
+                return f"-> executing bash search command: {command}{dir_info}\n"
+            return "-> executing bash search command\n"
+        case "semantic_search":
+            query = args.get("query", "")
+            project_id = args.get("project_id", "")
```

Ruff (0.15.1)

[error] Local variable `project_id` is assigned to but never used

Remove assignment to unused variable `project_id`

(F841)